tests/Fixer/ReturnNotation/ReturnAssignmentFixerTest.php

```diff
@@ -1063,5 +1063,66 @@ function Foo($food) {
             }
             ',
         ];
+
+        yield 'annotation for anonymous class' => [
```

Comment:
```suggestion
        yield 'anonymous class with attribute' => [
```
"Annotation" has strong link with phpDoc, while native PHP construct is called "attribute", so let's stick with it.

Reply:
fair point, I've updated the copy :+1: